docs/getting_started.md

```diff
@@ -1,17 +1,61 @@
 # Getting Started
 
-To get started with JetBot, just follow these steps
+## Step 1 - Pick your vehicle!
 
+To get started with JetBot, first pick your vehicle (hardware) you want to make.
 
-1. Get the parts from the [bill of materials](bill_of_materials.md).
+![](images/nvjetbot_vs_3rdparty.png)
 
-    > You can also start from one of the pre-bundled [third party kits](third_party_kits.md). 
+For details of NVIDIA-designed open-source JetBot hardware, check [Bill of Materials](bill_of_materials.md) page and [Hardware Setup](hardware_setup.md) page.
 
-2. Follow the [hardware setup](hardware_setup.md)
-2. Follow the [software setup](software_setup/docker.md)
-4. Run through the [examples](examples/basic_motion.md)
+To find kits available from 3rd parties, check [Third Party Kits](third_party_kits.md) page.
 
-After that, the rest is up to you! 
+
+## Step 2 - Setup your JetBot
+
+After you've decided which vehicle you want to build, you'll need to source the parts and follow the setup instructions. After that, you're ready to move on to the JetBot examples!
+
+![JetBot Getting Started Steps](images/steps_nvjetbot_vs_3rdparty.png)
+
+### DIY JetBot
+
+To get started with the DIY open-source JetBot, follow these steps.
+
+#### 1. Get the parts from the [bill of materials](bill_of_materials.md).
+
+#### 2. Follow the [hardware setup](hardware_setup.md)
+
+#### 3. Follow the [software setup](software_setup/docker.md)
+
+### Third Parth JetBot
+
+Third party JetBot kits may have different setup instructions from the DIY JetBot. To get started with a third party jetbot, we recommend referring to the third party kit's setup instructions for details on how to setup.
+
+## Step 3 - Follow the examples
+
+JetBot comes with many examples to teach robotics and AI basics. These give you the experience you need to begin exploring your own applications with JetBot.
+
+#### 1. [Basic Motion](examples/basic_motion.md) 
```

Comment:
I think we should make these actionable so the direction is clear.  "Follow the ..."

You can also just drop the headers ( ####) and collapse into list

1.  Follow the [Basic Motion](examples/basic_motion.md) example to learn how to program JetBot from your web browser with basic motor controls.
2.  Follow the [Teleoperation](examples/teleoperation.md) example to learn how to stream live images from JetBot and control JetBot with a gamepad attached to your laptop. 

Having the links to examples only appear as the section header is a bit counter intuitive.  This is because header links often simply refer to the header anchor (think the anchor tag on a github readme).  

To stick with section headers, the following format may work:

### Example 1 - Teleoperation

Follow the [basic motion](examples/basic_motion.md) example to learn how to program JetBot from your web browser with basic motor controls.

### Example 2 - Teleoperation

Follow the [teleoperation](examples/teleoperation.md) to learn how to stream live images from JetBot and control JetBot with a gamepad attached to your laptop.

If going the section headers route, a small image for each example might help justify the section header (since it's just a single sentence).